scipy/stats/_morestats.py

```diff
@@ -3957,14 +3960,17 @@ def directionalmean(samples, *, axis=0, normalize=True):
     Examples
     --------
     >>> from scipy.stats import directionalmean
-    >>> data = np.array([[0.8660254, 0.5, 0.], [0.8660254, -0.5, 0.]])
+    >>> data = np.array([[3, 4],    # first observation, 2D vector space
+    ...                  [6, -8]])  # second observation
```

Comment:
Attempts to clarify the distinction between the axis corresponding with separate observations (0) and the dimensionality of the vector space (-1).

Also, uses non-normalized vectors to show that (by default) the magnitudes won't matter.